### ISSUE TITLE
Import Vendors not present

### ISSUE THREAD

**Issue by LodsCory** [MicrosoftLearning/MB-800-Business-Central-Functional-Consultant#10]
# Module: 03
#### Step: 17

Description of issue
You are asked to import vendors V9001, V9003, and V9007 from the vendors list. These vendors do not exist.
Repro steps:

1. In step 17 you are asked to navigate to the vendors list page and multi-select vendors V9001, V9003, and V9007, but these vendors are not present.
1.
1.

**Comment by LaurieHeer** [comment]
Hello Cory. The lab steps are a bit vague, but there is a screenshot of the vendors to be imported at the top of Module 2: Lab 2 -  Exercise 1: Create and export a configuration package. There will be missing currencies that can be stubbed on import, or you can add them when you do company setup. I've submitted a sample import file and have updated the company setup steps in my pull request.

Hope this helps,
Laurie

**Comment by LodsCory** [comment]
Thank you Laurie. I can close this request and await yours if you'd like?

**Comment by LaurieHeer** [comment]
Cory,

I've attached the import to this email and have updated instructions/files
in a pull request. Keep in mind that the import file likely won't have the
additional currencies set up. Either take a look at my fork to see the
changes that I made to the instructions or follow the steps to let error
control create the missing codes. In real life you'd have to go back and
complete the currency exchange info, but it will fix the import.

Hope this helps,
Laurie Heer
